### ISSUE TITLE
Enable strict typing for all tests

### ISSUE THREAD

**Issue by LefterisJP** [rotki/rotki#842]
## Problem Definition

For backend tests we have an exception in setup.cfg here:
https://github.com/rotki/rotki/blob/b07063d35164f59b438a7385b3c72955a90e71d9/setup.cfg#L28-L32

We don't have strict typing check for all tests.

## Task

- Remove the lines shown above
- Properly type everything in tests so that linting passes again

**Comment by yashnirmal** [comment]
I am working on it

**Comment by yashnirmal** [comment]
Hi @LefterisJP can you check the PR once

**Comment by LefterisJP** [comment]
> Hi @LefterisJP can you check the PR once

As @yabirgb mentioned in the PR this is not the solution. One would need to actually type all tests properly

**Comment by yashnirmal** [comment]
Guys, can you help me with how I can do that?
Currently I am getting this error
![Screenshot from 2022-10-20 11-48-49](https://user-images.githubusercontent.com/78169683/196871371-31839f00-dc8e-4a59-a278-274cef87908b.png)

**Comment by yabirgb** [comment]
Hello @yashnirmal ! Did you modify the code in develop? If not make sure to do `pip install -r requirements.dev` to install the latest version of the linting packages. In develop you shouldn't get any error

**Comment by yashnirmal** [comment]
No. This error was also coming before I made any changes to the code.

**Comment by yabirgb** [comment]
@yashnirmal Then make sure that the version of flake8 is the same as in the `requirements_lint.txt` file.

```
(rotki-main) ➜  rotki git:(develop) ✗ pip freeze | grep flake8
flake8==5.0.4
flake8-bugbear==22.9.23
flake8-commas==2.1.0
flake8-comprehensions==3.10.0
flake8-debugger==4.1.2
flake8-executable==2.1.2
flake8-mutable==1.2.0
flake8-tuple==0.4.1
```

**Comment by AlexandrSozin** [comment]
Hi! I’d like to incrementally enable strict typing for a few small unit-test modules, starting with test_etherscan.py. Is this scope welcome?

**Comment by AlexandrSozin** [comment]
Hi! I’d like to incrementally enable strict typing for a few small unit-test modules, starting with test_etherscan.py. Is this scope welcome?
